py/server/tests/test_parquet.py

```diff
@@ -354,10 +354,12 @@ def test_dictionary_encoding(self):
         self.assertTrue((metadata.row_group(0).column(2).path_in_schema == 'someIntColumn') &
                         ('RLE_DICTIONARY' not in str(metadata.row_group(0).column(2).encodings)))
 
-    def test_dates(self):
-        dh_table = empty_table(20).update(formulas=[
+    def test_dates_and_time(self):
```

Comment:
Are we able to force pyArrow or similar to write millis or micros for time, so we can verify reading for those?

Reply:
Pyarrow uses micros by default, so was able to test that path.
For millis, I changed our writing code to write millis and then reran the tests. So couldn't add any tests in the pipeline for millis but tested it manually.
I can look into forcing it to use millis somehow to add tests for millis too.

Reply:
Added a test for this